SMPyBandits/Arms/Binomial.py

```diff
@@ -59,6 +59,12 @@ def draw_nparray(self, shape=(1,)):
         """ Draw a numpy array of random samples, of a certain shape."""
         return np.asarray(npbinomial(self.draws, self.probability, shape), dtype=float)
 
+    def set_mean_param(self, probability, draws=None):
+        if draws==None:
```

Comment:
Use `is None` to test equality with `None`.

Reply:
I will modify that